concepticondata/conceptlists.tsv

```diff
@@ -1,4 +1,5 @@
 ID	AUTHOR	YEAR	LIST_SUFFIX	ITEMS	TAGS	SOURCE_LANGUAGE	TARGET_LANGUAGE	URL	REFS	PDF	NOTE	PAGES	ALIAS
```

Comment:
Note that the change in user-rights in this file is to blame me, as my computer stupidly assigns 755 to some files, and I still did not figure out why.